examples/datadog-express-apm/host-agent/utils/response.js

```diff
@@ -0,0 +1,78 @@
+const agent = require('../agent').default
+/**
+ * @class Response
+ *
+ * @description class to handle all http response
+ */
+class ResponseHandler {
+  /**
+   *
+   * @description method to handle all success responses
+   *
+   * @param response Object
+   * @param statusCode String
+   * @param message String
+   * @param data Object | Array
+   *
+   * @return response JSON
+   */
+  static successResponse = (response, statusCode, message, data) => {
+    const responseBody = { status: 'success' }
+
+    if (message !== '') {
+      responseBody.message = message
+    }
+
+    if (data) {
+      responseBody.data = data
+    }
+
+    agent
+      .init()
+      .dogstatsd.increment(
+        `Commercetools/Client/Response/Success/${statusCode}`,
```

Comment:
Please update the names to the convention of datadog: https://docs.datadoghq.com/developers/guide/what-best-practices-are-recommended-for-naming-metrics-and-tags/